package.json

```diff
@@ -84,5 +84,10 @@
     "webpack": "^4.5.0",
     "webpack-cli": "^3.0.8",
     "webpack-dev-server": "^3.1.3"
+  },
+  "resolutions": {
+    "lodash": "4.17.21",
+    "hosted-git-info": "4.0.2",
+    "url-parse": "1.5.1"
```

Comment:
instead of exact version resolution, can we make these minor version match with the caret range specifier `^`?

Reply:
Ah yes. Did not know adding `^` before the versioning is called minor versioning matching. Learned something new today